nifi-commons/nifi-flow-encryptor/src/test/java/org/apache/nifi/flow/encryptor/StandardFlowEncryptorTest.java

```diff
@@ -89,7 +100,145 @@ public void testProcessNoEncrypted() {
         assertEquals(property, outputProperty);
     }
 
+    @Test
+    public void testProcessJson() throws JsonProcessingException {
+        final String password = "This is a bad password";
+        final String encryptedPassword = String.format(ENCRYPTED_FORMAT, inputEncryptor.encrypt(password));
+
+        final String sampleFlowJson = getSampleFlowJson(encryptedPassword);
+
+        final InputStream inputStream = new ByteArrayInputStream(sampleFlowJson.getBytes(StandardCharsets.UTF_8));
+        final ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
```

Comment:
Recommend declaring these inside a try-with-resources block for automatic closing.